.gitignore

```diff
@@ -14,6 +14,7 @@ CMakeUserPresets.json
 [Uu]ser/
 FrameCapture/**
 .DS_Store
+.automatedtesting.json
```

Comment:
This change is part of the change above to disable using the o3de-extras repo when building the stabilization branch and should not be merged over.

Reply:
Done, anything else?